crates/rspack_core/src/chunk_group.rs

```diff
@@ -15,21 +15,10 @@ use crate::{
 use crate::{ChunkLoading, ChunkUkey, Compilation};
 use crate::{LibraryOptions, ModuleIdentifier, PublicPath};
 
-#[derive(Debug, Clone)]
-pub struct SyntheticDependencyLocation {
-  pub name: String,
-}
-
-#[derive(Debug, Clone)]
-pub enum OriginLocation {
-  Real(DependencyLocation),
-  Synthetic(SyntheticDependencyLocation),
-}
-
 #[derive(Debug, Clone)]
 pub struct OriginRecord {
   pub module_id: Option<ModuleIdentifier>,
-  pub loc: Option<OriginLocation>,
+  pub loc: Option<DependencyLocation>,
```

Comment:
If we call `loc` here, should we just align every `range` with `loc` as well?

Reply:
We refer to `RealDependencyLocation` as range in `Dependency` because it serves as a `span`, utilizing its `start` and `end`, and can also be converted to `loc`.  

Here is the combination of `RealDependencyLocation` and `SyntheticDependencyLocation`, which is directly used as the `loc` for the `chunk` info, so it is not named as `range`.

Reply:
What's your opinion?

Reply:
That makes sense to me. Thanks for the explanation.